coinlib_flutter/windows/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# The Flutter tooling requires that developers have a version of Visual Studio
+# installed that includes CMake 3.14 or later. You should not increase this
+# version, as doing so will cause the plugin to fail to compile for some
+# customers of the plugin.
+cmake_minimum_required(VERSION 3.14)
+
+# Project-level configuration.
+set(PROJECT_NAME "coinlib_flutter")
+project(${PROJECT_NAME} LANGUAGES CXX)
+
+# Invoke the build for native code shared with the other target platforms.
+# This can be changed to accommodate different builds.
+add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../src" "${CMAKE_CURRENT_BINARY_DIR}/shared")
```

Comment:
This is where the CMakeLists.txt for Linux/Android is included. I'm not sure if this can be used for Windows too or if this should be removed in favour of Windows specific configuration placed into this file.

Reply:
My issues with implementing this today have mostly revolved around the disconnect between needing to run Flutter on the Windows host, but only being able to build in a WSL instance on Windows--what builds it can't run it, what runs it can't build it.

I may be able to build with native Windows tools.  Maybe there's a workaround or other approach I could take to accomplish the task.  But as for today, I wasn't able to complete the work as requested, sorry.

Reply:
No problem! I can leave it to you if this is something you'd like to continue with. If not, then that's fine, we can instead require users build the library separately. Either way, I appreciate the contribution. For Flutter users it is good to have some way to build for Windows, even if it's not as seamless as the other platforms.

With little time before Christmas, I can hold off merging. I'm happy to come back to this in the new year.

Reply:
I'll try again before the new year, but for now I have to work on integrating this currently-working (but manually-built) solution into other projects ([fusiondart](https://github.com/cypherstack/fusiondart) and [flutter_libsparkmobile](https://github.com/cypherstack/flutter_libsparkmobile/tree/testing)).

Late last night I thought that maybe I could use the Ubuntu WSL2 instance to navigate to the flutter project to be built/ran on Windows and run a `flutter run`/`flutter build`, which would cause CMake to cross-compile the lib before it will probably fail to complete the build or at least run it.  That might be a working approach.  But it's still a somewhat manual process...  Or I can try to get it building on Windows natively, though that comes with additional Visual Studio requirements, most likely.  That might be mostly-automatic except for requiring the right dependencies.  Would either of those approaches work, or would you prefer the latter?

Reply:
I imagine if users require the WSL2, it's almost as easy to run the build script than use an additional `flutter build` command? I think it would be reasonable to just keep the build script in that case. If it can instead be done natively through a single invocation of the flutter build process whilst requiring some dependencies I think that would be a good solution.

I'll be around for a while tomorrow and I might be around next Wednesday and/or Saturday. Though it might be that I'll have to take a look at any further changes in January.

Reply:
I was not able to get this completely working using just the CMakeLists.  `build_windows.dart` does work natively on Windows, however.